### DISCUSSION TITLE
How do I specify the target directory using the build command?

### DISCUSSION THREAD

**Discussion by KevinBlandy** [thuliteio/doks#400] [Q&A] [answered]
When building with `npm run build`, is there a parameter that specifies the output directory for static files?

thanks!

**Comment by h-enk** [comment, marked as answer]
You could use `-d, --destination` (see also [Hugo CLI](https://gohugo.io/getting-started/usage/#test-installation)), so e.g. run:

```
npm run build -- -d=outputdirectory
```

Or, update the build script:

```
"build": "exec-bin bin/hugo/hugo --gc --minify -d=outputdirectory",
```
